espanol/canudos.md

```diff
@@ -105,3 +105,6 @@ Conectó su guitarra al amplificador y se puso a tocar una versión de "Siempre
 A pero si no leiste el cuento del cuco, entonces significa que no tienes [miedo] (miedo/miedo.md)
 
 [Puso su mano derecha en su frente y se teletransportó al centro del ejército enemigo](pelea_epica/pelea_epica.md)
+
+Lo que nadie sabía, es que el consejero era un versado maestro instruído
+ en las nobles artes Jedi. [Dark_side](dark_side/darkside.md) [Light_side](light_side/light_side.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.